ui/src/app/components/update-spanner-config-form/update-spanner-config-form.component.ts

```diff
@@ -37,7 +37,11 @@ export class UpdateSpannerConfigFormComponent implements OnInit {
 
     this.fetch.setSpannerConfig(payload).subscribe({
       next: (res: ISpannerConfig) => {
-        this.snack.openSnackBar('Spanner Config updated successfully', 'Close', 5)
+        if (res.IsMetadataDbCreated) {
+          this.snack.openSnackBar('A new database has been created.', 'Close', 5)
```

Comment:
This snackbar might confuse the user on what database got created. We may want to improve the messaging

Reply:
Updated